drivers/rilmodem/sim.c

```diff
@@ -943,7 +944,8 @@ static void ril_query_pin_retries(struct ofono_sim *sim,
 			g_free(cbd);
 			CALLBACK_WITH_FAILURE(cb, NULL, data);
 		}
-	} else if (sd->vendor == OFONO_RIL_VENDOR_MTK) {
+	} else if (sd->vendor == OFONO_RIL_VENDOR_MTK ||
+			sd->vendor == OFONO_RIL_VENDOR_MTK2) {
```

Comment:
My testing has revealed that some quirk are needed for:

Only OFONO_RIL_VENDOR_MTK (like in g_ril_reply_parse_avail_ops())
Only OFONO_RIL_VENDOR_MTK2 (like in g_ril_unsol_parse_connected())
Both (most other quirks)

So we need both vendor tags.